packages/evo-marko/src/tags/evo-card/index.marko

```diff
@@ -0,0 +1,80 @@
+export type CardLayout = "horizontal" | "vertical";
+export type AspectRatio = "16:9" | "5:4" | "default";
+export interface CardElement extends Marko.HTML.P {
+    as?: string;
+}
+export interface CardOverlineElement extends Marko.HTML.P {
+    text?: string;
+}
+export interface Input extends Omit<Marko.HTML.Div, "title"> {
+    layout?: CardLayout;
+    aspectRatio?: AspectRatio;
+    overline?: Marko.AttrTag<CardOverlineElement>;
+    title?: Marko.AttrTag<CardElement>;
+    description?: Marko.AttrTag<CardElement>;
+    disabled?: boolean;
+    action?: Marko.AttrTag<CardElement>;
+    href?: string;
+    file?:
+        | File
+        | {
+              name: string;
+              type?: File["type"];
+              src?: string;
+          };
+}
+<const/{
+    class: inputClass,
+    layout,
+    href,
+    aspectRatio,
+    file,
+    overline,
+    title,
+    disabled,
+    description,
+    action,
+    ...htmlInput
+}=input>
+<const/elem=(!action ? (href ? "a" : "button") : "span")>
+
+<${elem}
+    class=[
+        "card",
+        aspectRatio === "5:4" && "card--aspect-5-4",
+        aspectRatio === "16:9" && "card--aspect-16-9",
+        layout === "horizontal" && "card--horizontal",
+        inputClass,
+    ]
+    disabled=elem === "button" && disabled
+    type=elem === "button" && "button"
+    href=elem === "a" && (disabled ? "" : href)
+    tabindex=disabled && -1
+    ...htmlInput>
+    <div class="card__media">
+        <img ...file>
```

Comment:
What happens here if this is actually a [File](https://developer.mozilla.org/en-US/docs/Web/API/File/File), as suggested in the `Input` type? I don't think it makes sense to spread that into `<img>`

Reply:
Should we do something similar that we do on [file-preview-card](https://github.com/eBay/evo-web/blob/main/packages/ebayui-core/src/components/ebay-file-preview-card/index.marko#L44-L59) where we check for the type and render accordingly?